src/plots/cartesian/layout_attributes.js

```diff
@@ -244,9 +244,9 @@ module.exports = {
         role: 'style',
         description: 'Determines whether or not the tick labels are drawn.'
     },
-    showspikes: {
+    showspike: {
```

Comment:
in `gl2d` it's already `showspikes` - which I think makes sense, as even if at any given time you only show one, there are many spikes you *could* show, ie one for each data point.